### ISSUE TITLE
post_textures.bat does not work correctly

### ISSUE THREAD

**Issue by Gamer-XP** [YoYoGames/GameMaker-Bugs#1588]
### Description

IDE: 2023.1.1.62
Runtime: 2023.1.1.81

I'm trying to use post_textures.bat to modify GM's generated texture pages. With a lot of issues I've managed to do some test changes, but there there are some weird issues that make it very hard to use:

1. My changes to the texture are never applied to the current build. I change the texture, game loads, in-game textures are still the way they were before I made the modification. GM should check for modifications after post-processign and update its internal data, or load that data AFTER post-process.

2. GM calls texture post-processing every single time, not only after it actually re-generates textures. In manual it says:
"This is executed after the textures for your game have been generated. You can use this step to optimise your game textures manually."
If that means "every time", then it's a bit confusing. I mean, you normally don't want to "optimize your game textures" multiple times in a row? If you want to keep current "post_textures" behaviour - maybe there should be either a flag or different event that is not called every time?

3. More of a feature request, but xml files for texture pages are lacking in info. What is missing:
- Sprite border value for the page
- Is this page a "separate texture page", which does not have border and crop rules applied to it
- How much pixels were cut from the sprite on each side
- Maybe width and height of the original sprite if possible.
Because that data is missing - I have to manually crop each sprite to see its actual position on the page.

### Expected Change

Change order in which post_textures.bat is called, and when it's called (optionally)

### Steps To Reproduce

[PostTexturesBug.zip](https://github.com/YoYoGames/GameMaker-Bugs/files/12445015/PostTexturesBug.zip)
(It's not a ContactUsPackage since it's older GM version, but I included relevant things)

1. Open and start the exampe project. You will see an image
2. Check the build log. There will be a line like: Checking textures at path: C:\Users\GamerXP\AppData\Roaming\GameMakerStudio2\Cache\GMS2CACHE\PostTextur_DD1C5FA6_498F280D\PostTexturesBug\Default\TextureGroups
3. Open the given path and find the texture page that contain sprite you saw. That texture page is supposed to have inverted colors, but in-game they were normal
4. Launch the game again. This time sprite will have inverted colors, but texture on disk got normal ones.

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

Other Release (Please put the exact version number in your Description)

### Which operating system(s) are you seeing the problem on?

Windows 10

### Which platform(s) are you having trouble building this project for?

Windows

### Are you running GameMaker from inside your Steam library?

No

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [X] I have included a small sample project

**Comment by jackerley** [comment]
Can you please submit each issue as a separate issue in github, otherwise it is very hard for us to track and assign workload? Thanks

**Comment by Gamer-XP** [comment]
Ok, I'll put 2 and 3 into separate issues then that will reference this one

**Comment by Gamer-XP** [comment]
Found a bigger issue after I tried building the project - changes are not carried to the build, which makes it literally unuseable unless, maybe, I manually unpack final build, replace texture there, then pack it back.
